### PR TITLE
Copy tokenizer files in each of their repo

### PR DESCRIPTION
**Pull request by sgugger**
# What does this PR do?

This PR cleans the maps in the tokenizer files to make sure each checkpoint has the proper tokenization files. This will allow us to remove custom code that mapped some checkpoints to special files (like BART using RoBERTa vocab files) and take full advantage of the versioning systems for those checkpoints. All checkpoints changed have been properly copied in the corresponding model repos in parallel.

For instance, to accomodate the move on the fast BART tokenizers, the following commits have been on the model hub:
- in [facebook/bart-base](https://huggingface.co/facebook/bart-base/commit/c2469fb7e666a5c5629a161f17c9ef23c85217f7)
- in [facebook/bart-large](https://huggingface.co/facebook/bart-large/commit/22fa33834dccc11df99c4fc5fcc96c67f806dfdb)
- in [facebook/bart-large-mnli](https://huggingface.co/facebook/bart-large-mnli/commit/6a35c499ad1087bad8d9c348a05b1fa10c5ad47d)
- in [facebook/bart-large-cnn](https://huggingface.co/facebook/bart-large-cnn/commit/18614750f248f300641757e8e44e6afce801d664)
- in [facebook/bart-large-xsum](https://huggingface.co/facebook/bart-large-xsum/commit/96ea79a741cd376cdc8a740b225330773da151f0)
- in [yjernite/bart_eli5](https://huggingface.co/yjernite/bart_eli5/commit/38797dd2ef06f5542c6f7db853518703f6b3da21)

In the PR I've also uniformized the way the maps are structured across models, to make it easier to alter (and ultimately remove) them in the future via automatic scripts.

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Fantastic work! Pinging @thomwolf and @julien-c for review, as this is the first step of removing archive maps from the repository.

**Comment by julien-c** [comment]
Love it!

Maybe a good practice to link to a sample of the related commits on hf.co: for instance here https://huggingface.co/facebook/bart-base/commit/c2469fb7e666a5c5629a161f17c9ef23c85217f7

**Comment by sgugger** [comment]
I think I did around 50 of them in various repos to move all the tokenizers files, so a bit hard to keep track of all of them.

**Comment by julien-c** [comment]
Yep just link one, or a small sample
